charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -8,6 +8,10 @@ image:
   tag: ""
   pullPolicy: IfNotPresent
 
+# -- Custom labels to add into metadata
+customLabels: #{}
```

Comment:
I think custom labels should be empty for the default values for the chart

Reply:
done